public/data/ShorunTech.json

```
@@ -0,0 +1,54 @@
+{
+    "name": "ShorunTech",
+    "type": "personal",
+    "bio": "Self-Taught Fulk-Stack Web Developer,Youtuber,Content Creator,Open Source Enthusiast!",
+    "avatar": "https://github.com/ShorunTech.png",
+    "links": [{
+            "name": "Follow me on GitHub",
+            "url": "https://github.com/ShorunTech",
+            "icon": "github"
+        },
+        {
+            "name": "Follow me on Twitter",
+            "url": "https://twitter.com/shoruntech",
+            "icon": "twitter"
+        },
+        {
+            "name": "Follow me on Linkedin",
+            "url": "https://www.linkedin.com/in/shorunke-gbenga-95295a197/",
+            "icon": "linkedin"
+        },
+        {
+            "name": "Follow me on Youtube",
+            "url": "https://www.youtube.com/channel/UCCEAthZ8ntnhz1umBAKlsRA",
+            "icon": "youtube"
+        }
+    ],
+    "milestones": [{
+            "title": "Frontend Developer",
+            "date": "Febuary 2019",
+            "icon": "dollar",
+            "color": "yellow",
+            "description": "Started freelancing",
+            "url": "https://www.eleven22creativestudio.com/"
+        },
+        {
+            "title": "YouTuber",
+            "date": "December 2021",
+            "icon": "dollar",
+            "color": "red",
+            "description": "Create Tutorial Generally On Programming",
+            "url": "https://www.youtube.com/channel/UCCEAthZ8ntnhz1umBAKlsRA"
+
+        },
+        {
+            "title": "Full-Stack Web Developer",
+            "date": "December 2022",
+            "icon": "dollar",
+            "color": "green",
+            "description": "fulltime Developer",
+            "url": "shorun-tech.vercel.app/"
```

It's always a good practice to specify the `https` extension in such projects. 

```suggestion
            "url": "https://shorun-tech.vercel.app/"
```